README.md

```diff
@@ -1,8 +1,20 @@
 # changelist
 
-Prepare an automatic changelog from GitHub pull requests. For example, see
+Changelist helps you write better release notes by automating as much of the
+process as possible. For example, see
 https://github.com/scientific-python/changelist/blob/main/CHANGELOG.md.
 
+**Features**
+
+- Compile a list of pull requests, code authors and reviewers between any two
+  valid Git objects (refs).
+- Categorize pull requests into sections based on GitHub labels.
+- Point it at any repository on GitHub. No need to clone or checkout a
+  repository locally, a Python package is all that's needed.
+
+We recommend to treat the generated document as a first draft to build
```

Comment:
I don't agree with this. If we had bigger projects with more full-time developers, I would support this. It is great to have really nice release notes, but I would prefer to see more focus on new features and improvements.

Reply:
How I see this sentence is that it emphasizes that one needs to read the generated changelog before pushing a release rather than blindly assuming all is fine. Just like one does (or should do) with the github generated release notes.

Reply:
Yes, that's precisely what I was trying to convey here. :)